treeherder/webapp/api/performance_data.py

```diff
@@ -889,6 +902,45 @@ def _get_signatures(self, repository_name, framework, interval, no_subtests):
         signatures = self._get_signatures_values(signatures)
         return signatures
 
+    def _create_graph_links(
+        self,
+        base_repo_name,
+        new_repo_name,
+        base_revision,
+        new_revision,
+        framework,
+        timeRange,
+        signature,
+    ):
+        highlighted_revision_key = 'highlightedRevisions'
+        time_range_key = 'timeRange'
+        seriesKey = 'series'
+
+        highlighted_revisions_params = [
+            (highlighted_revision_key, base_revision[:12]),
+            (highlighted_revision_key, new_revision[:12]),
+        ]
+
+        graph_link = 'graphs?%s' % urlencode(highlighted_revisions_params)
+
+        if new_repo_name == base_repo_name:
+            # if repo for base and new are not the same then make diff
+            # series data one for each repo, else generate one
+            repo_value = ','.join([new_repo_name, signature, '1', framework])
+            graph_link = graph_link + urlencode({seriesKey: repo_value})
```

Comment:
```suggestion
            graph_link = graph_link + '&' + urlencode({seriesKey: repo_value})
```
You are missing the '&' here